scrapers_next/ny/committees.py

```diff
@@ -50,21 +51,26 @@ def process_page(self):
 
         try:
             chairs = CSS(".chair-info").match(self.root)
+
         except SelectorError:
             raise SkipItem("skipping committee without full information")
 
         # in case there are co-chairs
-        num_chairs = len(chairs)
-
-        for chair in chairs:
+        for index, chair in enumerate(chairs):
             chair_name = CSS(".comm-chair-name").match_one(chair).text_content().strip()
+
+            # index for the current chair in the list of chairs
+            actual_index = index + 1
```

Comment:
Since this is going to happen on every step, could we just say `{index + 1}` below and skip the extra variable?
And also maybe explain why we're iterating past our current index?

Reply:
Yeah, used that when I was testing, it broke somehow. I'll retry it again since everything is working good now. Will add the comment too